client/containers/clusters/index.js

```diff
@@ -12,262 +12,270 @@ import Service from '../service';
 import ContainerInstance from '../container-instance';
 
 export default class ClustersContainer extends Component {
-  static childContextTypes = {
-    awsConfig: React.PropTypes.object,
-  };
-
-  constructor(props, context) {
-    super(props, context)
-    this.state = {
-      error: null,
-      clusters: [],
-      services: [],
-      containerInstances: [],
-      activeClusterArn: null,
-      activeServiceArn: null,
-      awsConfig: {},
+    static childContextTypes = {
+        awsConfig: React.PropTypes.object,
     };
-  }
-
-  /** 
-   * Pass through 
-  */
-  getChildContext() {
-    return {
-      awsConfig: this.state.awsConfig,
-    };
-  }
-
-  /**
-   * Render.
-   */
-
-  render() {
-    const activeClusterArn = this.getActiveClusterArn();
-    const isLoading = !!this.state.error || !!this.state.services.length;
-    return (
-      <Page>
-        <Sidebar
-          clusters={this.state.clusters}
-          activeClusterArn={activeClusterArn}
-          searchTerm={this.state.searchTerm}
-          setSearchTerm={::this.setSearchTerm}
-          selectCluster={::this.setActiveCluster} 
-          onRefresh={::this.fetchData}
-        />
-
-        <Loader loaded={isLoading} color="#3cc76a">
-          {this.renderError()}
-          <ServiceList
-            services={this.state.services}
-            searchTerm={this.state.searchTerm}
-            activeClusterArn={activeClusterArn} />
-          <ContainerInstanceList
-            containerInstances={this.state.containerInstances}
-            activeClusterArn={activeClusterArn}
-          />
-        </Loader>
-        {this.renderServiceSheet()}
-        {this.renderContainerInstanceSheet()}
-      </Page>
-    )
-  }
-
-  /**
-   * Get the active cluster ARN.
-   */
-
-  getActiveClusterArn() {
-    if (this.state.activeClusterArn) {
-      return this.state.activeClusterArn;
+
+    constructor(props, context) {
+        super(props, context)
+        this.state = {
+            error: null,
+            clusters: [],
+            services: [],
+            containerInstances: [],
+            activeClusterArn: null,
+            activeServiceArn: null,
+            awsConfig: {},
+            isGrid: true
+        };
+        this.toggleGrid = this.toggleGrid.bind(this);
+    }
+
+    toggleGrid() {
+        this.setState(prevState => ({
+            isGrid: !prevState.isGrid
+        }));
+    }
```

Comment:
Add a newline between function declarations, plus, the other functions in this project have documentation comments. Add one here to be consistent with the existing code.